orm/src/test/java/org/hibernate/search/test/analyzer/inheritance/AnalyzerInheritanceTest.java

```diff
@@ -53,12 +56,12 @@ public void testBySearch() throws Exception {
 		tx = s.beginTransaction();
 
 
-		QueryParser parser = new QueryParser( "name", s.getSearchFactory().getAnalyzer( SubClass.class ) );
-		org.apache.lucene.search.Query luceneQuery = parser.parse( "name:Proca\u00EFne" );
+		QueryParser parser = new QueryParser( "name", TestConstants.keywordAnalyzer );
+		org.apache.lucene.search.Query luceneQuery = parser.parse( "name:John" );
 		FullTextQuery query = s.createFullTextQuery( luceneQuery, SubClass.class );
 		assertEquals( 1, query.getResultSize() );
 
-		luceneQuery = parser.parse( "name:Procaine" );
+		luceneQuery = parser.parse( "name:Doe" );
```

Comment:
I'm a bit worried about this change. The StandardAnalyzer that might be used as default would have the exact same behavior. I really think using an analyzer filtering the non ASCII characters is better. I think it's feasible, isn't it?